processing/create_mosaic.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+#
+# XX
+#
+# USAGE:
+#   ${SCT_DIR}/python/bin/python XX
+#
+#   Example:
+#   ${SCT_DIR}/python/bin/python create_mosaic.py -p t2_unf.nii.gz,t2_milan.nii.gz -c t2 -ax_sag 4 -n 3
+#
+# DEPENDENCIES:
+#   SCT
+#
+
+import os
+import argparse
+import sys
+import numpy as np
+import matplotlib.pyplot as plt
+
+from spinalcordtoolbox.image import Image
+from spinalcordtoolbox.centerline.optic import detect_centerline
+import sct_utils as sct
+
+
+def main():
+    # create temporary folder with intermediate results
+    tmp_folder = sct.TempFolder(verbose=1)
```

Comment:
why are you working in a temp dir and copy the input files there, instead of simply load them in memory?